### PR TITLE
FEAT: Added flag -std=c99 to enable building on older versions of gcc (4.8.5 specifically)

### PR DESCRIPTION
**Pull request by chafey**
See https://github.com/filecoin-project/neptune-triton/pull/9 for reference.  

NOTE: I was unable to test due to problems getting getting futhark installed in my environment

### CODE REVIEW

**Comment by athas** [comment]
Futhark only requires C99, but it probably does not matter.

**Comment by porcuquine** [comment]
@chafey Is there a reason on your end to use C11 rather than C99? You also mentioned some other flags which would reduce warnings. If you can add those, that would be great.

@Erk- I have another build-related change which I will submit in another PR. Maybe we can bundle these both into a next release.

**Comment by Erk-** [comment]
@porcuquine sure

**Comment by chafey** [comment]
C99 is fine, I used C11 because it worked and is newer.  After thinking about this more, it might be best to use gnu11 since that is how it is successfully being compiled today (gcc changed to gnu11 with version 5, before that it was gnu89).  I couldn't make any progress on flags to reduce warnings since I couldn't get futhark installed to test.

**Comment by Erk-** [comment]
It should just be c99 I think, the others are not standardised and specific to the gnu c compiler I belive.

**Review by Erk- — APPROVED** [review]

**Comment by Erk-** [comment]
Can I get you to rebase on master?

**Comment by chafey** [comment]
rebased

File: src/static/build.rs [hunk 1]
```diff
@@ -37,10 +39,22 @@ fn main() {
...
+        {
+            cc::Build::new()
+                .file("./lib/a.c")
+                .flag("-fPIC")
+                .flag("-std=c99")
+                .flag("-framework")
+                .flag("OpenCL")
+                .shared_flag(true)
+                .compile("a");
+        }
```
Comment (Erk-) [inline src/static/build.rs]: This should not be here, it was just removed in #14

**Review by Erk- — CHANGES_REQUESTED** [review]

File: src/static/build.rs (hunk 1 above)
Comment (chafey) [reply src/static/build.rs]: ugh sorry, i need a better diff tool :(

**Comment by Erk-** [comment]
Thanks 😄

**Pull request merged**